### DISCUSSION TITLE
🟠 Router | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#624] [Announcements]
# 🟠 Router | Angular Challenges

Challenge 17 is about testing the router

https://angular-challenges.vercel.app/challenges/testing/17-router/

**Comment by idembele70** [comment]
The command `npx nx component-test testing-router-outlet` didn't worked.
I have to run this one below from the location /angular-challenges
```sh
npx nx component-test testing-router
```
